db/src/main/resources/db/migration/V1_058__delete_claimant_table.sql

```diff
@@ -0,0 +1,15 @@
+update claim  set
```

Comment:
Running this query again to catch any claims created by old versions of the service during blue/green deployment